bigquery/google/cloud/bigquery/job.py

```diff
@@ -447,7 +487,8 @@ def cancelled(self):
         :rtype: bool
         :returns: False
         """
-        return False
+        return (self.error_result is not None
+                and self.error_result.get('reason') == 'stopped')
```

Comment:
1. No.
2. It's from the list of reasons above. I meant to add a comment there. Done in incoming commit.